src/global/types.h

```diff
@@ -2051,6 +2051,12 @@ typedef struct SAMPLE_DATA {
 
 #pragma pack(pop)
 
+typedef struct PICTURE {
```

Comment:
Do we care about the packing on this?

Reply:
Not really, this structure belongs to us only